pkg/solana/chainwriter/transform_registry_test.go

```diff
@@ -1,59 +1,109 @@
 package chainwriter_test
 
 import (
+	"context"
 	"testing"
 
 	"github.com/gagliardetto/solana-go"
+	ccipconsts "github.com/smartcontractkit/chainlink-ccip/pkg/consts"
 	"github.com/smartcontractkit/chainlink-ccip/pkg/types/ccipocr3"
 	"github.com/smartcontractkit/chainlink-common/pkg/utils/tests"
 	"github.com/stretchr/testify/require"
 
+	"github.com/smartcontractkit/chainlink-solana/pkg/monitoring/testutils"
 	"github.com/smartcontractkit/chainlink-solana/pkg/solana/chainwriter"
+	"github.com/smartcontractkit/chainlink-solana/pkg/solana/client"
+	clientmocks "github.com/smartcontractkit/chainlink-solana/pkg/solana/client/mocks"
 )
 
-func Test_CCIPCommitTransform(t *testing.T) {
+func Test_CCIPExecuteTransform(t *testing.T) {
```

Comment:
nit: make the Test_[name] pattern actually match the real function.
So Test_CCIPExecuteArgsTransform and Test_CCIPCommitAccountTransform